src/app/shared/pipes/userDisplay.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+interface UserInfos {
+  login: string,
+  firstName :string|null,
+  lastName :string|null,
+}
+
+@Pipe({ name: 'userDisplay' })
+export class UserDisplayPipe implements PipeTransform {
```

Comment:
I don't really like the "User Display" term... 
To be consistent with the other angular pipe, the better name would probably "user" and the class name "UserPipe".

Reply:
fixed in PR #197